install_config/storage_examples/privileged_pod_storage.adoc

```diff
@@ -0,0 +1,202 @@
+= Mount Volumes on Privileged Pods
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap:
+
+== Overview
+
+The following provides a method for mounting persistent volumes to pods attached
+to the *privileged scc*.  For the purpose of this guide, GlusterFS will be the
+sample use-case though this method can be adapted to any
+link:../install_config/persistent_storage/index.html[supported storage plugin].
+
+=== Prerequisites
+* A gluster volume exists and *glusterfs-fuse* is installed on all hosts
+* Definitions written for GlusterFS link:../persistent_storage/persistent_storage_glusterfs.html#creating-gluster-endpoints[endpoints and service],
+link:../persistent_storage/persistent_storage_glusterfs.html#gfs-creating-persistent-volume[persistent volume]
+and
+link:../architecture/additional_concepts/storage.html#persistent-volume-claims[persistent volume claim]
+* A user with the
+link:../admin_guide/manage_authorization_policy.html#managing-role-bindings[cluster-admin] role binding.
+For this guide, that user is called `admin`
+
+[[create-basic-user]]
+== Create a basic-user
+
+Create the basic-user by logging in.
```

Comment:
We don't often mention the "basic-user" role, so it may not mean much to the reader. Maybe markup "basic-user" here in bold like it is in https://docs.openshift.com/enterprise/latest/architecture/additional_concepts/authorization.html#roles and link the word to that topic/section to provide some context if needed.